cfgmgr/vxlanmgr.cpp

```diff
@@ -1,4 +1,7 @@
 #include <algorithm>
+#include <regex>
+#include <sstream>
+#include <string>
```

Comment:
already in header file

Reply:
I add duplicated header file, because I want to decouple the dependency of this module with other modules, which can avoid the compile error of this module if these header files are removed in other files.